src/app/saga.ts

```diff
@@ -298,12 +299,6 @@ export function* handleDeepLink(action: OpenDeepLink) {
   const { isSecureOrigin } = action
   Logger.debug(TAG, 'Handling deep link', deepLink)
 
-  const walletAddress = yield* select(walletAddressSelector)
```

Comment:
nit: i wonder if we should leave this here and continue to expect the wallet address as a param in the jumpstartClaim saga. only the jumpstartClaim saga is technically needing the wallet address right now, so i understand why you made the change. 

it's been annoying in the past for deep links to be processed when wallet address is not available. for example, the opened_via_invite_url event ended up being unhelpful because the walletAddress was undefined and we couldn't match inviters and invitees. the logic in the wallet has changed now where this saga shouldn't be triggered unless the walletAddress is available, but that is done in a hook and perhaps nice to fail early here if something goes wrong?

Reply:
but failing early will mean we don't show the user any feedback about the error, right?

we could add a generic alert, but failing inside the specific handler means we could communicate what specific situation we are processing, and what error we've encountered -- depending on the handler features

Reply:
hmm yes, i guess i can see it both ways. 

failing early here = we assert all deep links should be handled after the wallet is initialized and it does not matter whether the deep link strictly needs the wallet address in the logic. even if the logic succeeds, our instrumentation will be broken becuase the wallet address is missing.

failing contextually = we have more information about where it failed, but we will allow silent instrumentation failures. 

i have a preference for the first scenario, because i cannot think of a scenario where we'd want to process a deep link when a wallet is not initialised?

Reply:
I agree. There may be a logic requiring a wallet address not _directly_ in a deep link handler, down the user journey, like when link handler fires a navigate event.